app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -398,6 +398,19 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
       ]);
     }
 
+    if (siteSettings.enable_upcoming_changes) {
```

Comment:
Looks like this in the sidebar when enabled:

<img width="348" height="170" alt="image" src="https://github.com/user-attachments/assets/4de36267-b5e3-4cb7-8e72-fb438ed396da" />